eegrasp/eegrasp.py

```diff
@@ -71,27 +81,35 @@ def gaussian_kernel(self, x, sigma=0.1):
 
         Notes
         -----
-        This function is supposed to be used in the PyGSP2 module but
-        is repeated here since there is an error in the available version
-        of the toolbox (03/04/2024 dd/mm/yyyy)
+        This function is supposed to be used in the PyGSP2 module but is repeated here since there
```

Comment:
This bug doesn't exist in PyGSP anymore. We should remove the function (but in a different issue/pr).